docs/api/apiv3/endpoints/schemas.apib

```diff
@@ -40,16 +40,17 @@ Only one of the links (`allowedValues`, `allowedValuesSchemas`) will exist for a
 
 ## Local Properties
 
-| Property          | Description                                                                        | Type     | Default |
-|:-----------------:| ---------------------------------------------------------------------------------- | -------- | ------- |
-| name              | Human readable name of the property as it could be displayed in a UI               | String   |         |
-| type              | The data type of the properties values                                             | MetaType |         |
-| minLength         | The value of the property must at least contain the specified amount of characters | Integer  | 0       |
-| maxLength         | The value of the property must at most contain the specified amount of characters  | Integer  | ∞       |
-| regularExpression | The value of the property must match the given regular expression (if not null)    | String   | null    |
-| required          | If true this property is not nullable                                              | Boolean  | true    |
-| writable          | If false it is not allowed to **change** the properties value                      | Boolean  | true    |
-| location          | If present, contains a reference to the location of the property in the JSON       | String   |         |
+| Property          | Description                                                                        | Type         | Default |
+|:-----------------:| ---------------------------------------------------------------------------------- | ------------ | ------- |
+| name              | Human readable name of the property as it could be displayed in a UI               | String       |         |
+| type              | The data type of the properties values                                             | MetaType     |         |
+| minLength         | The value of the property must at least contain the specified amount of characters | Integer      | 0       |
+| maxLength         | The value of the property must at most contain the specified amount of characters  | Integer      | ∞       |
+| regularExpression | The value of the property must match the given regular expression (if not null)    | String       | null    |
+| required          | If true this property is not nullable                                              | Boolean      | true    |
+| writable          | If false it is not allowed to **change** the properties value                      | Boolean      | true    |
+| location          | If present, contains a reference to the location of the property in the JSON       | String       |         |
+| description       | If present, contains a formattable, human readable description                     | Formattable  |         |
 
 All of the above properties that do not have a default value *must* be present in the schema.
```

Comment:
The addition of the `_meta` group should also be described in the `forms.apib` documenation as it will be a general pattern that clients should expect to happen in all payload objects.